.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+stages:
+  - build
+
+trigger_testbed:
+  stage: build
+  image:
+    name: registry.gitlab.sensirion.lokal/sensirion/docker/docker-ubuntu:18.04-1.2.0
+  tags:
+    - docker
+    - linux
+  script:
+  - >
+    curl
+    --request POST
+    --form ref=test-pipeline
```

Comment:
Should be master right?

Reply:
yes, thanks. A leftover from the chicken and egg problem ;)